tests/unittests/core/io/database_test.py

```diff
@@ -68,9 +67,9 @@ def test_valid_attributes(self, create_db_instance):
         assert readonly_database.username == database.username
```

Comment:
is_connected, port, username and password does not make sense for PickledDB. We could drop.

Reply:
I removed these checks